package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "darwin",
+  "version": "0.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "install-fe": "./.scripts/frontend/install.sh",
+    "install-be": "./.scripts/backend/install.sh",
+    "build-fe": "./.scripts/frontend/build.sh",
+    "build-be": "./.scripts/backend/build.sh",
+    "test-fe": "./.scripts/frontend/test.sh",
+    "test-be": "./.scripts/backend/build.sh",
```

Comment:
Why not just `cd src && npm run x`?
As long as there is no need to do bashing I would encourage to keep it simple (and cross-platform compatible)

Reply:
And wouldn't it be nice to declare these scripts in the corresponding package.json of the package instead of here?

Reply:
Switched to basic npm scripts
The app-shell orchestrates the corresponding packages to run specific tasks, but they are defined inside each package. 🤔 
Would you suggest to directly call them from the pipeline and skip the app-shell part?

Reply:
Don't really care - it's nice to be able to call them from the project root, therefore I'd keep them in the root.